doc/source/whatsnew/v0.16.1.txt

```diff
@@ -156,3 +158,9 @@ Bug Fixes
 - Bug where dividing a dataframe containing values of type ``Decimal`` by another ``Decimal`` would raise. (:issue:`9787`)
 - Bug where using DataFrames asfreq would remove the name of the index. (:issue:`9885`)
 - Changed caching in ``AbstractHolidayCalendar`` to be at the instance level rather than at the class level as the latter can result in unexpected behaviour. (:issue:`9552`)
+=======
+- Fixed bug where ``DataFrame.plot()`` raised an error when both ``color`` and ``style`` keywords were passed and there was no color symbol in the style strings (:issue:`9671`)
+
+
+_Biweight Midcorrelation: http://download-v2.springer.com/static/pdf/95/chp%253A10.1007%252F978-3-642-39678-6_14.pdf?token2=exp=1429037148~acl=%2Fstatic%2Fpdf%2F95%2Fchp%25253A10.1007%25252F978-3-642-39678-6_14.pdf
```

Comment:
This link doesn't work for me. If you're going to link a paper (though would rather use wikipedia), using doi.org is probably the most reliably way to ensure it works.